Bio
---
Tony Kabilan Okeke
Philadelphia, PA
Email: tonykabilanokeke@gmail.com
Phone: 646-659-4319
Location: Philadelphia, PA
LinkedIn: [Tony Kabilan Okeke](https://linkedin.com/in/t-k-o)
GitHub: [Kabilan108](https://github.com/Kabilan108)
Portfolio: kabilan108.github.io
Role: Data Scientist/Bioinformatician

Education
---------
Drexel University, Philadelphia, PA
M.Sc. in Biomedical Engineering, Anticipated Graduation: June 2024
B.Sc. in Biomedical Engineering, Anticipated Graduation: June 2024
- Cumulative GPA: 4.0/4.0
- Specializations: Bioinformatics, Neuroengineering, Bioimaging

Technical Skills
----------------
- **Programming Languages:** Python, R, Bash, C++, MATLAB, SQL, AWK, Git, PHP, JavaScript
- **Libraries \& Frameworks:** TensorFlow, Keras, PyTorch, Scikit-learn, Flask, FastAPI, Shiny, Dash, Streamlit
- **Bioinformatics Tools:** Seurat, SquidPy, CellProfiler, Columbus, Loupe Browser
- **Wet Lab Assays:** PCR, qPCR, ELISA, Western Blot, DNA/RNA/Protein Extraction, NGS Library Preparation

Professional Experience
-----------------------
### IVIVT-Non-Clinical Safety, GlaxoSmithKline (GSK), Computational Biology Co-op, Collegeville, PA
- **Term:** April 2023 - September 2023
    Elevated spatial transcriptomics data analysis by integrating advanced machine learning models and 10X Visium technology, enhancing quality control and cellular annotation.
    - Developed software solutions to seamlessly integrate internally generated spatial transcriptomics data with publicly available single-cell data, advancing comprehensive cellular analysis.
    - Orchestrated the internal deployment of PFRED software on GSK's high-performance computing system, streamlining RNA-seq investigations into species-specific cellular responses.
    - Engineered a Python-based bioinformatic tool for automating the import and analysis of tissue-specific gene expression data, accelerating the oval cell hyperplasia tox study.
    - Partnered with the eSTAR working group to build a neural network model that predicts molecular initiating events in liver carcinogenicity, leveraging rat transcriptomics studies.
    - Employed UMAP and t-SNE techniques to cluster HESI compounds, and developed machine learning models for biomarker identification within each cluster, enhancing targeted assessments in carcinogenicity.
    - Spearheaded the development of an interactive spatial transcriptomics analysis platform for GeoMx technology, supporting future acquisition plans and facilitating in-depth, interactive reporting.

### IVIVT-Non-Clinical Safety, GlaxoSmithKline (GSK), Computational Biology Co-op, Collegeville, PA
- **Term 1:** April 2022 - September 2022
    - Engineered an interactive web app for statistical analysis in high-content imaging toxicology via Python's `Dash` and `Flask`; scaled computations using `Dask` and deployed on `RStudio Connect`.
    - Authored a Python package encapsulating statistical, visualization, and machine learning tools for high-content imaging.
    - Implemented machine learning algorithms (Decision Trees, Random Forests, SVM) for biomarker discovery in high-content imaging datasets.
    - Enhanced image quality using `Noise2Void` deep learning techniques, contributing to more precise image segmentation pipelines in `CellProfiler`.
    - Implemented feature extraction pipelines for high-content imaging data using `CellProfiler` and `Columbus`.

### Invenio Lab, Hospital of the University of Pennsylvania, Immunology Research Assistant, Philadelphia, PA
- **Term:** March 2021 - August 2022
    - Standardized protocols and performed assays for DNA, RNA, and protein isolation; prepared NGS libraries.
    - Employed unsupervised machine learning to interpret clinical and multi-omic data, presented via `Jupyter` notebooks.
    - Created Python and R scripts for DNA methylation analysis from Illumina microarray data.
    - Conducted comprehensive data analysis on post-cardiopulmonary bypass patients, including differential methylation and pathway enrichment.

### Zhou Lab, Children's Hospital of Philadelphia, Undergraduate Research Intern, Philadelphia, PA
- **Term:** May 2020 - June 2021
    - Collaborated in the development of open-source R packages for DNA methylation analysis.
    - Validated package functionalities using publicly available GEO datasets.

Projects
--------
### Meddibia
- **Technologies:** Python, TensorFlow, Flask, Flutter
- **Duration:** March 2023
  - Received the Collaborative Team Award at the 2023 Philly CodeFest.
  - Engineered a machine learning pipeline using pre-trained models like VGG16 and EfficientNet for skin condition diagnosis, achieving approximately 70% accuracy in image-based classification.
  - Utilized GPT-3 for symptom identification from user descriptions and trained a Random Forest classifier to predict corresponding diseases with 87% accuracy. Integrated GPT-3 to offer comprehensive disease information to users.
  - Designed a cross-platform app with a Flask API backend deployed on Heroku and a Flutter-based frontend, ensuring seamless user experience across multiple devices.

### MLGO: Deep Learning for Predicting GO Enrichment
- **Technologies:** Python, TensorFlow
- **Duration:** September 2022
  - Developed an R pipeline for preprocessing RNA-Seq datasets and conducting differential expression analysis using data from GEO and DEE2.
  - Crafted a neural network for predicting Gene Ontology (GO) enrichment in differentially expressed genes from RNA-Seq experiments.
  - Trained an autoencoder model to reconstruct log fold-change values in RNA-Seq data, using the latent space as input for a neural network to predict GO terms.
  - Employed TensorBoard and Keras Tuner for real-time monitoring of model performance and hyperparameter tuning.

### CaBiD: Cancer Biomarker Discovery Tool
- **Technologies:** Python, Flask, Qt
- **Duration:** September 2022
  - Created a web application and GUI for exploring gene expression variations across multiple types of cancer.
  - Curated and preprocessed datasets from GEO (Gene Expression Omnibus) and CuMiDa (Curated Microarray Database), stored in an SQLite database.
  - Analyzed key differences in gene expression between healthy controls and tumoral samples across various cancer types.

### ELISA Analysis Tool
- **Technologies:** R, Shiny
- **Duration:** September 2021
  - Processed Optical Density (OD) values from microplate readers using tidyverse packages in R.
  - Developed an R script for fitting OD values to a 5-Parameter logistic regression model for ELISA standard curves, enabling the estimation of unknown sample concentrations.
  - Constructed an interactive web application for ELISA curve fitting using the RShiny framework.


Publications
------------
- Laudanski, Krzysztof, et al. "Unbiased analysis of temporal changes in immune serum markers in acute COVID-19 infection with emphasis on organ failure, anti-viral treatment, and demographic characteristics." \href{https://www.frontiersin.org/articles/10.3389/fimmu.2021.650465/full}{Frontiers in Immunology 12 (2021): 650465.}
- Laudanski, Krzysztof, et al. "Longitudinal urinary biomarkers of immunological activation in covid-19 patients without clinically apparent kidney disease versus acute and chronic failure." \href{https://www.nature.com/articles/s41598-021-99102-5.pdf?proof=tr}{Scientific Reports 11.1 (2021): 19675}.
- Laudanski, Krzysztof, et al. "Dynamic changes in central and peripheral neuro-injury vs. Neuroprotective serum markers in COVID-19 are modulated by different types of anti-viral treatments but do not affect the incidence of late and early strokes." \href{https://doi.org/10.3390/biomedicines9121791}{Biomedicines 9.12 (2021): 1791.}
- Laudanski, Krzysztof, et al. "A disturbed balance between blood complement protective factors (FH, ApoE) and common pathway effectors (C5a, TCC) in acute COVID-19 and during convalesce." \href{https://doi.org/10.1038/s41598-022-17011-7}{Scientific Reports 12.1 (2022): 13658.}
- Laudanski, Krzysztof, et al. "Persistent Depletion of Neuroprotective Factors Accompanies Neuroinflammatory, Neurodegenerative, and Vascular Remodeling Spectra in Serum Three Months after Non-Emergent Cardiac Surgery." \href{https://doi.org/10.3390/biomedicines10102364}{Biomedicines 10.10 (2022): 2364}.

Conference Abstracts
--------------------
- 174: LONGITUDINAL CHANGES OF NEURO-SPECIFIC SERUM PROTEINS IN COVID-19 PATIENTS \href{https://doi.org/10.1097/01.ccm.0000807020.73561.ed}{Society of Critical Care Medicine 51st Critical Care Congress, April 2022}.
- 181: PATTERNS OF URINARY BIOMARKERS OF IMMUNOLOGIC ACTIVATION AND SEPSIS IN PATIENTS WITH COVID-19 \href{https://doi.org/10.1097/01.ccm.0000807048.74105.59}{Society of Critical Care Medicine 51st Critical Care Congress, April 2022}.

Awards and Honors
-----------------
- HESI GTTC Professional Development Award, Spring 2022, $1000
- Philly CodeFest Collaborative Team Award, Spring 2022, $5000
- Drexel University Dean's List, Fall 2019 - Present

Service and Leadership
----------------------
- **Drexel Computational Design**
  - Co-founder \& Vice President, March 2021 - Present
  - Project Manager, May 2022 - Present
- **Tau Beta Pi Honor Society**
  - Member, December 2021 - Present
- **Drexel Society of Artificial Intelligence**
  - Member, September 2022 - Present
